docs/05-plugin-development.md

```diff
@@ -236,11 +243,12 @@ See [custom module meta-data](guide/en/#custom-module-meta-data) for how to use
 You can use [`this.getModuleInfo`](guide/en/#thisgetmoduleinfomoduleid-string--moduleinfo--null) to find out the previous values of `moduleSideEffects`, `syntheticNamedExports` and `meta` inside this hook.
 
 #### `watchChange`
-Type: `(id: string) => void`<br>
+Type: `(id: string, isDeleted: boolean) => void`<br>
```

Comment:
Making `isDeleted` a single boolean parameter seems a little limiting in case more flags end up being useful later. Could it be `{ isDeleted: boolean }` instead?

Reply:
Yep, I can do it.

Reply:
Done!